traffic_ops/install/bin/postinstall

```diff
@@ -25,10 +25,17 @@ for arg in "$@"; do
 	esac
 done
 
+PATH+=:/usr/libexec/
+python_bin="$(command -v {python,platform-python}{3{8,.8,6,.6,},2{7,.7,},} | head -n1)"
```

Comment:
I realize this is recreating the supported versions from the old shebang, but why not include 3.7?

Reply:
No reason not to. We could include 3.9, too

Reply:
Including Python 3.7 and 3.9 in 3f272e6f2f